auth/auth.go

```diff
@@ -310,12 +322,23 @@ func (myAuth *Auth) AwsS3Credentials(level string, bucket string, prefix string)
 //   * auth:azure-table-access:<account>/<table>
 //
 // See http://docs.taskcluster.net/auth/api-docs/#azureTableSAS
-func (myAuth *Auth) AzureTableSAS(account string, table string) (*AzureSharedAccessSignatureResponse, *tcclient.CallSummary, error) {
+func (myAuth *Auth) AzureTableSAS(account, table string) (*AzureSharedAccessSignatureResponse, *tcclient.CallSummary, error) {
 	cd := tcclient.ConnectionData(*myAuth)
 	responseObject, callSummary, err := (&cd).APICall(nil, "GET", "/azure/"+url.QueryEscape(account)+"/table/"+url.QueryEscape(table)+"/read-write", new(AzureSharedAccessSignatureResponse), nil)
 	return responseObject.(*AzureSharedAccessSignatureResponse), callSummary, err
 }
 
+// Returns a signed URL for AzureTableSAS, valid for the specified duration.
+//
+// Required scopes:
+//   * auth:azure-table-access:<account>/<table>
+//
+// See AzureTableSAS for more details.
+func (myAuth *Auth) AzureTableSAS_SignedURL(account, table string, duration time.Duration) (*url.URL, error) {
```

Comment:
Using camel casing and underscores just struck me off guard.  Not that it's bad, I'm more just trying to understand what's common in Go.  Same goes for my other comments about understanding error strings, return types, etc. so please don't take it as me nit picking! I'm just trying to learn :)